js/mapview.js

```diff
@@ -58,9 +58,9 @@ fetchData() {
 render() {
   super.render();
   let app = document.getElementById('app');
-  app.innerHTML = '';
-  let sideBarEl = document.createElement('div');
-  sideBarEl.setAttribute('id', 'sidebar');
+  // app.innerHTML = '';
```

Comment:
It seems like this would completely prevent switching views without having leftovers of the previous view?